packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -181,20 +181,24 @@ class IconLabelButton extends StatelessWidget {
             : null,
         child: outlined
             ? OutlinedButton(
-                style: denseAwareOutlinedButtonStyle(
-                  context,
-                  minScreenWidthForTextBeforeScaling,
-                ),
+                style: minScreenWidthForTextBeforeScaling != null
+                    ? denseAwareOutlinedButtonStyle(
+                        context,
+                        minScreenWidthForTextBeforeScaling!,
+                      )
+                    : null,
```

Comment:
instead of explicitly passing null here, can we just continue to have `denseAwareOutlinedButtonStyle` properly handle a null value for `minScreenWidthForTextBeforeScaling`? same comment for below

Reply:
denseAwareOutlinedButtonStyle requires non null minScreenWidthForTextBeforeScaling

Reply:
We can change that. `denseAwareOutlinedButtonStyle ` calls `includeText`, which handles a null value for `minScreenWidthForTextBeforeScaling`

Reply:
done